specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putComputeOneSkuQuotaRequest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardFSv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "name": {
+          "value": "standardFSv2Family"
```

Comment:
>standardFSv2Family [](http://example.com/codeflow?start=20&length=18)

Do we really need this? Not a good user experiance if the user needs to specify the resource name in the URI as well as in the body.

Reply:
Yes, the name property looks duplicate.
We can remove the value part, as it's already included in name.
It was based on - https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html?q=usages
"name": {
        "value": "Storage",
        "localizedValue": "Storage"
      }

Reply:
You can keep it if that's ARM team's recommendation. But your RP should validate that the two names match in a PUT/PATCH request.

Reply:
The RP should match the name passed in parameter matches the request name.value.